Add book model and configure admin panel accordingly

theTytoAlba opened the pull request:
Hi!

We needed a book model to keep track of which comment belongs to which book etc.

Here you can see the book objects, they have isbn (necessary), and optionally they have title and author. If we need anything else we can add more fields later.
<img width="1280" alt="screen shot 2017-11-13 at 09 44 11" src="https://user-images.githubusercontent.com/7192055/32713176-1dd3368c-c859-11e7-861c-cb8cf36f4847.png">

Here you can see we can select a book for a comment.
<img width="1239" alt="screen shot 2017-11-13 at 09 50 54" src="https://user-images.githubusercontent.com/7192055/32713151-f7cd74de-c858-11e7-8669-30f5c52b1caa.png">

Here you can see we can filter comments by users or books.
<img width="1263" alt="screen shot 2017-11-13 at 09 53 12" src="https://user-images.githubusercontent.com/7192055/32713141-e7c1dc2e-c858-11e7-8254-1884eab93580.png">

sevgicansalih commented on bookbotproject/conversationtree/admin.py:
```diff
@@ -18,7 +18,14 @@
-                    list_display=('__unicode__', 'user', 'comment'),
+                    list_filter=('user', 'book'),
+                    list_display=('__unicode__', 'user', 'comment', 'book'),
                                        list_display_links=('__unicode__',),)
```
Indentations are important :) It is not a big deal, you can make the change next time you commit.

theTytoAlba replied:
You are right! Fixed it now. ^^

sevgicansalih commented on bookbotproject/conversationtree/admin.py:
```diff
@@ -18,7 +18,14 @@
                     ,)
+admin.site.register(Book,
+                    admin.ModelAdmin,
+                    list_filter=('author', ),
```
As I understood, we need author field to search a book, If I am correct, I have some ideas,
Filtering books by only using author may not be sufficient. We may think to add a new field:"genre" to our structure, In addition to adding a new field, we can use this field as a filtering field in here. If I misunderstood anything, please correct me by opening a discussion in Issues.(This place may not be the best to have a discussion)

theTytoAlba replied:
I agree with you. I am adding the genre field now, since it will be nullable, we can choose not to use it in the worst case scenario. :)

theTytoAlba commented:
Hi! I addressed your comments and added the genre field to books, books can be filtered by genre as well as author now. See here:
<img width="1258" alt="screen shot 2017-11-15 at 11 50 25" src="https://user-images.githubusercontent.com/7192055/32826979-8bc98bd4-c9fb-11e7-9006-2fe45cc76d17.png">

Please take another look! :)

agoz17 commented:
Good work Irmak as always. I just want to add this little point. I think we may need the title as necessary. I can't think scenario which a book with only isbn is commented in our system.
